### PR TITLE
[core] Bumped `@aptos-connect/wallet-adapter-plugin` to 2.0.2

### PR DESCRIPTION
**Pull request by hardsetting**
... and exposed `preferredWalletName` in the adapter config

https://github.com/user-attachments/assets/86cc05ae-57ed-46cf-967b-6dfa81b1a6e8

### CODE REVIEW

File: packages/wallet-adapter-core/src/WalletCore.ts [hunk 1]
```diff
@@ -87,7 +87,12 @@ export type IAptosWallet = AptosStandardWallet & Wallet;
 
 export interface DappConfig {
   network: Network;
+  /** @deprecated */
   aptosConnectDappId?: string;
+  aptosConnect?: {
+    dappId?: string;
```
Comment (0xmaayan) [inline packages/wallet-adapter-core/src/WalletCore.ts]: nit: can we add a doc string above `aptosConnect` prop explaining each of the property options?

**Review by 0xmaayan — APPROVED** [review]

File: packages/wallet-adapter-core/src/WalletCore.ts [hunk 2]
```diff
@@ -82,12 +82,16 @@ import {
 import { GA4 } from "./ga";
 import { WALLET_ADAPTER_CORE_VERSION } from "./version";
 import { aptosStandardSupportedWalletList } from "./AIP62StandardWallets/registry";
+import type { AptosConnectWalletConfig } from "@aptos-connect/wallet-adapter-plugin";
 
 export type IAptosWallet = AptosStandardWallet & Wallet;
 
 export interface DappConfig {
   network: Network;
+  /** @deprecated */
```
Comment (0xmaayan) [inline packages/wallet-adapter-core/src/WalletCore.ts:91]: if this is deprecated, what should we put here? https://github.com/aptos-labs/aptos-wallet-adapter/blob/main/apps/nextjs-example/src/components/WalletProvider.tsx#L36

Comment (hardsetting) [reply packages/wallet-adapter-core/src/WalletCore.ts:91]: we could put
"aptosConnect: { dappId }"
instead, but wanted to keep that value there also to make sure it still works even if deprecated

Comment (0xmaayan) [reply packages/wallet-adapter-core/src/WalletCore.ts:91]: ohh I see, 
```
export interface AptosConnectWalletConfig extends Omit<ACDappClientConfig, 'defaultNetworkName'> {
  network?: Network;
  preferredWalletName?: string;
}
```

and `ACDappClientConfig` has a `dappId` prop

**Pull request merged**